### PR TITLE
Add mic to FireFly Lora adapter

### PR DESCRIPTION
**Pull request by BobClaerhout**
Signed-off-by: Bob Claerhout <claerhout.bob@gmail.com>

### CODE REVIEW

File: adapters/lora-vertx/src/test/java/org/eclipse/hono/adapter/lora/providers/FireflyProviderTest.java [hunk 1]
```diff
@@ -58,6 +62,17 @@ public void extractTypeFromLoraUplinkMessage() {
         assertEquals(LoraMessageType.UPLINK, type);
     }
 
+    /**
+     * Verifies that the extracted mic is true
+     */
+    @Test
+    public void extractRssiFromLoraMessage() {
```
Comment (sophokles73) [inline adapters/lora-vertx/src/test/java/org/eclipse/hono/adapter/lora/providers/FireflyProviderTest.java]: sure that this should be called extractRssiFromLoraMessage and not extractMicFromLoraMessage ?

File: adapters/lora-vertx/src/main/java/org/eclipse/hono/adapter/lora/LoraConstants.java [hunk 2]
```diff
@@ -55,6 +55,7 @@
     public static final String CODING_RATE = "coding_rate";
     public static final String FREQUENCY = "frequency";
     public static final String FRAME_COUNT = "frame_count";
+    public static final String APP_PROPERTY_MIC = "mic";
```
Comment (sophokles73) [inline adapters/lora-vertx/src/main/java/org/eclipse/hono/adapter/lora/LoraConstants.java]: can you please move this up to the other `APP_PROPERTY_`s

**Comment by sophokles73** [comment]
@BobClaerhout there are also two checkstyle violations which you need to fix for the build to succeed.

File: adapters/lora-vertx/src/main/java/org/eclipse/hono/adapter/lora/LoraConstants.java (hunk 2 above)
Comment (BobClaerhout) [reply adapters/lora-vertx/src/main/java/org/eclipse/hono/adapter/lora/LoraConstants.java]: done

File: adapters/lora-vertx/src/test/java/org/eclipse/hono/adapter/lora/providers/FireflyProviderTest.java (hunk 1 above)
Comment (BobClaerhout) [reply adapters/lora-vertx/src/test/java/org/eclipse/hono/adapter/lora/providers/FireflyProviderTest.java]: you're right, fixed

**Comment by BobClaerhout** [comment]
> @BobClaerhout there are also two checkstyle violations which you need to fix for the build to succeed.

Should be fixed now

**Pull request merged**